gpkit/constraints/relax.py

```diff
@@ -105,8 +105,7 @@ def __init__(self, constraints, include_only=None, exclude=None):
         exclude = frozenset(exclude) if exclude else frozenset()
         include_only = frozenset(include_only) if include_only else frozenset()
         substitutions = KeyDict(constraints.substitutions)
-        constants, _, _ = parse_subs(constraints.varkeys,
-                                     constraints.substitutions)
+        constants = parse_subs(constraints.varkeys, constraints.substitutions)
```

Comment:
I changed `parse_subs` to have different returns. Pretty gross, definitely revertable.